app/views/smart_proxies/welcome.html.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for(:title, _("Smart Proxies")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("sitemap", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Smart Proxies') %></h1>
+  <p><%= _("The Smart Proxy is a project which provides a RESTful API to various subsystems.
+           The Smart Proxy provides an easy way to add or extended existing subsystems, via
```

Comment:
https://github.com/theforeman/foreman/pull/4475#discussion_r112650447 remains unaddressed, unsure why the focus on plugins is here and not Foreman. This should explain what Foreman uses it for and why smart proxies are listed here, not about the capabilities of another project.